superbit_lensing/medsmaker/superbit/astro_config/sextractor.mock.config

```diff
@@ -73,7 +73,7 @@ PIXEL_SCALE      0              # size of pixel in arcsec (0=use FITS WCS info)
 
 #------------------------- Star/Galaxy Separation ----------------------------
 
-SEEING_FWHM      0.3            # stellar FWHM in arcsec
+SEEING_FWHM      0.265          # stellar FWHM in arcsec
```

Comment:
Yes, it's a rough guess. As far as I can tell, the `SEEING_FWHM` parameter is just used for the sextractor `CLASS_STAR` estimation. As noted in the docs, `CLASS_STAR` has been superseded by `SPREAD_MODEL`, but `SPREAD_MODEL` requires a PSF model... so there is some non-linearity here.

Reply:
This might be a good area for an eventual enhancement, by the way. Maybe we can have a "master issue" with desired enhancements, like not hardcoding sky_sigma and turning the `PiffExtender` class into an adaptor.